smaug/python/ops/array_ops.py

```diff
@@ -344,3 +344,28 @@ def check_and_add_layout_transform(name, op, input_tensors):
       input_tensors[i] = reorder(input_tensors[i], expected_layoutset.layouts)
   return input_tensors
 
+def padding(input_tensor, padding_size, name="padding"):
+  """Construct a tensor by padding a given tensor.
+
+  Args:
+    input_tensor: Input tensor.
+    padding_size: A list that contains number of values padded to each dimension.
```

Comment:
I think this docstring needs to be more explicit: padding_size is in the format of {dim0_begin, dim0_end, dim1_begin, dim1_end, ...}.